docs/CLI.md

```diff
@@ -71,7 +71,7 @@ The test environment used for all tests. This can point to any file or node modu
 
 Alias: `-e`. Use this flag to show full diffs instead of a patch.
 
-### `--findRelatedTests=<listOfSourceFiles>`
+### `--findRelatedTests <listOfSourceFiles>`
```

Comment:
yep I like that.  Should keep the Source though so people don't think its for listing test files